ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.11.yaml

```diff
@@ -56,7 +56,7 @@ tests:
   interval: 960h
   steps:
     cluster_profile: vsphere
-    workflow: cucushift-installer-rehearse-vsphere-ipi-capi
+    workflow: cucushift-installer-rehearse-vsphere-ipi-disconnected-techpreview
```

Comment:
For disconnected scenario, we need to use cluster profile "vsphere-discon", which will choose correct disconnected network for cluster installation on vSphere, suggest to add new section here which is the same as one in above 4.10 job.  https://github.com/openshift/release/pull/29844/files#diff-ea05105b2925414db38c1251c56f44730fa9183edb850cef6d8b6fddb53f1a92R74-R78

Reply:
Got it, thanks! as we already validate it in 4.10 job, so we don't need to revalidate it in 4.11 as they are the same for the two versions, so change it to `cucushift-installer-rehearse-vsphere-ipi` as `cucushift-installer-rehearse-vsphere-ipi-capi` workflow does not exist.